.github/workflows/codeql-analysis.yml

```
@@ -21,25 +21,25 @@ defaults:
+    name: Check conditional workflows and jobs
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
     runs-on: ubuntu-latest
     outputs:
-      java: ${{ steps.changes.outputs.java }}
-      themes: ${{ steps.changes.outputs.themes }}
-      js-adapter: ${{ steps.changes.outputs.js-adapter }}
+      java: ${{ steps.conditional.outputs.codeql-java }}
+      themes: ${{ steps.conditional.outputs.codeql-themes }}
+      js-adapter: ${{ steps.conditional.outputs.codeql-js_adapter }}
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)